build.sbt

```diff
@@ -330,7 +330,7 @@ lazy val json4s: ProjectMatrix = (projectMatrix in file("json/json4s"))
     name := "tapir-json-json4s",
     libraryDependencies ++= Seq(
       "org.json4s" %%% "json4s-core" % Versions.json4s,
-      "org.json4s" %%% "json4s-jackson" % Versions.json4s % Test,
+      "org.json4s" %%% "json4s-jackson" % Versions.json4s,
```

Comment:
for sure we want to keep the jackson dependency in the `Test` scope :)

Reply:
ah, sure, it should be directly copied into test dependencies of tapir-openapi-docs..